index.html

```diff
@@ -4,9 +4,42 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Raj</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,400;0,600;0,700;1,100;1,300;1,400;1,600;1,700&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="css/style.css">
+    
 </head>
 <body>
-    
+    <header>
+        <nav class="menu">
+            <div>
+                <a href="#" class="logo">My Logo</a>
+            </div>
+            <div class="menu-icon">
+                <img src="/img/Icon - Menu.png" alt="">
+            </div>
+        </nav>
+    </header>
+
+    <main>
+        <div class="container">
+            <div class="content">
+                <h1 class="my-name">I’m Mariane <br> Glad to see <br> you!</h1>
+                <p class="my-name-sub">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <a href="#" class="connect-me">LET'S CONNECT</a>
+            </div>
+            <div class="social-media-icon">
+                <img src="/img/social icons/twiter icon.svg" alt="">
+                <img src="/img/social icons/github icon.svg" alt="">
+                <img src="/img/social icons/icon.svg" alt="">
+                <img src="/img/social icons/Linkedin icon.png" alt="">
+                <img src="/img/social icons/medium icon.svg" alt="">
+            </div>
```

Comment:
- [ ] Kindly make sure your social links are inside and unordered list `ul` and not a `div`
- [ ] And also grab all the `img` inside a `anchor <a></a>` as they are links to your social media and each one of them inside a `li`  
`<ul>`
`<li> <a href="'> <img /></a></li>`
`</ul>`